src/test/java/com/wikia/webdriver/elements/mercury/components/discussions/common/PostEntity.java

```diff
@@ -136,7 +136,7 @@ public boolean hasEditedBySection() {
           .getAttribute(CLASS_ATTRIBUTE)
           .contains(POST_EDITED_BY_CLASS_NAME);
     } catch (NoSuchElementException e) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1166)